pypesto/result/optimize.py

```diff
@@ -162,20 +164,27 @@ def __getitem__(self, index):
     def __len__(self):
         return len(self.list)
 
-    def append(
-        self,
-        optimizer_result: OptimizerResult,
-    ):
+    def append(self, opt_result: OptimizationResult, to_sort: bool = True):
         """
-        Append an optimizer result to the result object.
+        Append an OptimizerResult or an OptimizeResult to the result object.
 
         Parameters
         ----------
-        optimizer_result:
-            The result of one (local) optimizer run.
+        opt_result:
+            The result of one or more (local) optimizer run.
+        to_sort:
+            Boolean used so we only sort once when appending an
+            optimize_result.
```

Comment:
Can rename `to_sort` to `sort`

Reply:
thought that would be misleading together with self.sort()

Reply:
Or, less misleading to use the same name :see_no_evil: